test_framework/terraform/digitalocean/do.tf

```diff
@@ -13,11 +14,11 @@ resource "digitalocean_ssh_key" "do-ssh-key" {
 resource "digitalocean_droplet" "longhorn-tests" {
   count    = 4
   image    = "ubuntu-18-04-x64"
-  name     = "longhorn-tests-0${count.index}"
+  name     = "${var.hostname_prefix}-0${count.index}"
   region   = "nyc3"
   size     = "s-4vcpu-8gb"
   ssh_keys = ["${digitalocean_ssh_key.do-ssh-key.fingerprint}"]
-  tags     = ["longhorn-tests", "k8s-controller"]
```

Comment:
What's the reason for removing `k8s-controller` here?

Reply:
incorrect tagging, it will tag all instances with `k8s-controller`

Reply:
It's also not used, and will not affect running tests jobs

Reply:
If it's not related, put it in a separate commit then.